nservicebus/bridge/configuration_failed-messages_bridge_[2,).partial.md

```diff
@@ -0,0 +1,16 @@
+### Handling of failed messages
```

Comment:
should we call this "Error queue" to be in line with the above heading of "audit queue"?

Reply:
Done